README.md

```diff
@@ -1,53 +1,186 @@
-# Harmony Browse Image Generator (HyBIG) backend service.
+# Harmony Browse Image Generator (HyBIG).
 
 This Harmony backend service is designed to produce browse imagery, with
 default behaviour to produce browse imagery that is compatible with the NASA
 Global Image Browse Services ([GIBS](https://www.earthdata.nasa.gov/eosdis/science-system-description/eosdis-components/gibs)).
 
+This means that defaults for images are selected to match the visualization
+generation requirements and recommendations put forth in the GIBS Interface
+Control Document (ICD), which can be found on [Earthdata
+Wiki](https://wiki.earthdata.nasa.gov/display/GITC/Ingest+Delivery+Methods)
+along with [additional GIBS
+documentation](https://nasa-gibs.github.io/gibs-api-docs/).
+
+HyBIG creates paletted PNG images and associated metadata from GeoTIFF input
+images. Scientific parameter raster data as well as RGB[A] raster images can
+be converted to browse PNGs.  These browse images undergo transformation by
+reprojection, tiling and coloring to seamlessly integrate with GIBS.
+
+### Reprojection
+
+GIBS expects to receive images in one of three Coordinate Reference System (CRS) projections.
+
+| Region      | Code      | Name                                                      |
+|-------------|-----------|-----------------------------------------------------------|
+| north polar | EPSG:3413 | WGS 84 / NSIDC Sea Ice Polar Stereographic North          |
+| south polar | EPSG:3031 | WGS 84 / Antarctic Polar Stereographic                    |
+| global      | EPSG:4326 | WGS 84 -- WGS84 - World Geodetic System 1984, used in GPS |
```

Comment:
I was _so good_ and didn't make a comment on the spacing within this table. But it made me chuckle to see that you tweaked it anyway.

Reply:
emacs did it for me on accident when I hit return in there somehow. but it was bugging me too